universe/interface.go

```diff
@@ -552,6 +552,44 @@ type FederationLog interface {
 	LogNewSyncs(ctx context.Context, addrs ...ServerAddr) error
 }
 
+// ProofType is an enum that describes the type of proof which can be stored in
+// a given universe.
+type ProofType int
+
+const (
+	// ProofTypeNone indicates that all proofs should be excluded from the
+	// universe.
+	ProofTypeNone ProofType = iota
+
+	// ProofTypeIssuance indicates that issuance proofs should be included
+	// in the universe.
+	ProofTypeIssuance
+
+	// ProofTypeTransfer indicates that transfer proofs should be included
+	// in the universe.
+	ProofTypeTransfer
+)
+
+// String returns a human-readable string representation of the sync proof types
+// enum.
+func (t ProofType) String() string {
+	return []string{"none", "issuance", "transfer"}[t]
```

Comment:
Right, good catch. Since in Goland a type isn't actually an enum, nothing is stopping you from creating `foo := ProofType(1234)`. So maybe just go with the usual switch that returns `unknown proof type <%d>` as the default?